pumps/elasticsearch.go

```diff
@@ -355,12 +370,53 @@ func (e *ElasticsearchPump) Init(config interface{}) error {
 		e.log.Info("Index will have date appended to it in the format ", e.esConf.IndexName, "-YYYY.MM.DD")
 	}
 
+	if e.esConf.UseSSL {
+		e.log.Debug("Getting SSL Certificates")
+		tlsConfig, err := setTLSConfig(e.esConf.SSLCertFile, e.esConf.SSLKeyFile, e.esConf.SSLInsecureSkipVerify)
+		if err != nil {
+			e.log.Fatal("Failed to set TLS config: ", err)
+		}
+		e.esConf.tlsConfig = tlsConfig
+	}
+
 	e.connect()
 
 	e.log.Info(e.GetName() + " Initialized")
 	return nil
 }
 
+// setTLSConfig sets the TLS config for the pump
+func setTLSConfig(SSLCertFile, SSLKeyFile string, SSLInsecureSkipVerify bool) (*tls.Config, error) {
```

Comment:
This function is called `SetTLSConfig`, but it doesn't set anything. Instead let's rename it `GetTLSConfig`

Reply:
I choose that name because it was setting the tlsConfig field of the esConf struct. But now that it's been deleted, I think `GetTLSConfig` is a better name for this function!